### PR TITLE
@ normalize-by-median.py : multiple same named input files will now rais...

### PR DESCRIPTION
**Pull request by elmbeech**
...e a warning. 
bug #908

### CODE REVIEW

**Comment by mr-c** [comment]
Jenkins, okay to test

**Comment by mr-c** [comment]
Jenkins, ok to test

**Comment by elmbeech** [comment]
- [ ] Is it mergeable?
- [ ] Did it pass the tests?
- [ ] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage with `make clean diff-cover`
- [ ] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [ ] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [ ] Is it documented in the ChangeLog?
  http://en.wikipedia.org/wiki/Changelog#Format
- [ ] Was a spellchecker run on the source code and documentation after
  changes were made?

**Comment by elmbeech** [comment]
cleaned out commit for #908

File: scripts/normalize-by-median.py [hunk 1]
```diff
@@ -223,6 +223,18 @@ def main():  # pylint: disable=too-many-branches,too-many-statements
...
     report_fp = args.report
 
+    # check for similar filenames
+    filenames = []
+    for pathfilename in args.input_filenames:
+        filename = pathfilename.split('/')[-1]
+        if (filename in filenames):
+            print >>sys.stderr, "WARNING: At least two input files are named \
+%s . (The script normalize-by-median.py can not handle this, only one .keep \
+file for one of the input file will be generated.)" % filename
```
Comment (mr-c) [inline scripts/normalize-by-median.py]: "one of the input file_s_ will be generated."

**Comment by mr-c** [comment]
@elmbeech Congratulations on your first commit to the khmer project! Your name will be included in the release notes for the next version and you'll be listed amongst our other contributors in the next software release paper.

**Pull request merged**